### PR TITLE
Remove community meeting requirements from governance docs

### PR DESCRIPTION
**Pull request by mrbobbytables**
The community meeting has not been an effective means of communicating important information or updates for a few years now, and is no longer occurring on a regular cadence (and may be removed altogether soon in the future).

As such it should be removed as a community group governance requirement.

/committee steering
/assign @kubernetes/steering-committee 

/hold for comments

### CODE REVIEW

File: committee-steering/governance/README.md
Comment (cblecker) [inline committee-steering/governance/README.md:46]: I don't think slack is a sufficient way to announce large scale changes. Perhaps we just leave it to the mailing list?

**Comment by MadhavJivrajani** [comment]
xref https://github.com/kubernetes/community/issues/7051

**Comment by nimbinatus** [comment]
Thanks for the xref, @MadhavJivrajani!

File: committee-steering/governance/README.md
Comment (mrbobbytables) [reply committee-steering/governance/README.md:46]: I'm good with that, I meant it more as an optional thing to mention the change in there. the mailing list should ALWAYS get the notifications.

Comment (mrbobbytables) [reply committee-steering/governance/README.md]: just changed and/or to and (optionally)

**Comment by tpepper** [comment]
This makes me sad but,
/approve
knowing there's parallel work to revitalize the sharing of information across our project parts internally and with our external stakeholders/users, which used to be the hallmark of the meeting when it was more active.

**Review by palnabarun — APPROVED** [review]
/approve

**Comment by mrbobbytables** [comment]
At this time we have enough approvals from Steering to merge, but I will leave this PR open till 1/31 for any remaining comments.

**Review by Rahul-Prasad-07 — APPROVED** [review]

**Review by cpanato — COMMENTED** [review]
/approve

**Pull request merged**
